docs/templates/toc.yml

```diff
@@ -107,6 +107,8 @@
         uid: bigframes.ml.llm.PaLM2TextGenerator
       - name: PaLM2TextEmbeddingGenerator
         uid: bigframes.ml.llm.PaLM2TextEmbeddingGenerator
+      - name: GeminiTextGenerator
```

Comment:
Should be sorted it in alphabetic order.

Reply:
Done